bin/kyuubi

```diff
@@ -102,7 +102,7 @@ if [[ -n ${YARN_CONF_DIR} ]]; then
   KYUUBI_CLASSPATH="${KYUUBI_CLASSPATH}:${YARN_CONF_DIR}"
 fi
 
-cmd="${RUNNER} ${KYUUBI_JAVA_OPTS} -cp ${KYUUBI_CLASSPATH} $CLASS"
+cmd="${RUNNER} ${KYUUBI_JAVA_OPTS} -cp ${KYUUBI_CLASSPATH} $CLASS ${@:2}"
```

Comment:
it also allows `bin/kyuubi` without command. I think we can shift the consumed args and pass `$@` here

Reply:
@pan3793 Thanks, I missed that case.

Why do we need to support running Kyuubi as a daemon with `bin/kyuubi` without `start` command? At first look, it would be more expected to me to print help message if no command provided by user. I can create a PR if you think it is a good change.

Also I found another interesting case. As a user, I would expect `restart` to launch Kyuubi with the same configs was used to launch Kyuubi with `start` command initially. Not sure how to manage it automatically, so in current implementation `restart` command requires users to provide the same `--conf` options was used with `start` command.

Reply:
> Why do we need to support running Kyuubi as a daemon with `bin/kyuubi` without `start` command?

this is a behavior from the very beginning, let's keep it to avoid breaking the user experience

> ... so in current implementation `restart` command requires users to provide the same `--conf` options was used with `start` command.

I see, and it is reasonable to me.

Reply:
@pan3793 I added commit to address your initial suggestion. Please, have a look if it looks good.

---
> > Why do we need to support running Kyuubi as a daemon with `bin/kyuubi` without `start` command?
> 
> this is a behavior from the very beginning, let's keep it to avoid breaking the user experience
> 

Right. It is make sense to leave it as is, at least before Kyuubi 2.0